modules/source/mail/config/mail.xml

```diff
@@ -34,7 +38,7 @@
 			store-uri="${protocol}://${username:}:${password:}@${host}:${port}/${folder}"
```

Comment:
No since that means file access as well.  I believe we have modules already that accept a spring resource location for additional properties.  Not sure if we ever supported something inline to specify properties.